src/Microsoft.IdentityModel.Xml/DsigSerializer.cs

```diff
@@ -212,6 +228,29 @@ private void ReadIssuerSerial(XmlReader reader, KeyInfo keyInfo)
          }
 
         /// <summary>
+        /// Reads the "RSAKeyValue" element conforming to https://www.w3.org/TR/2001/PR-xmldsig-core-20010820/#sec-RSAKeyValue.
+        /// </summary>
+        /// <param name="reader">A <see cref="XmlReader"/> positioned on a <see cref="XmlSignatureConstants.Elements.RSAKeyValue"/> element.</param>
+        /// <param name="keyInfo">The <see cref="KeyInfo"/> to hold the RSAKeyValue.</param>
+        private void ReadRSAKeyValue(XmlReader reader, KeyInfo keyInfo)
```

Comment:
It would be better to return a RSAKeyKeyValue

Reply:
This is currently not the way that the other read methods in DSigSerializer work.
I created a separate issue to address this problem:
https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/issues/766